addons/website_sale/static/src/snippets/s_dynamic_snippet_products/000.scss

```diff
@@ -35,8 +35,52 @@
 }
 
 .o_dynamic_product_hovered {
-    transition: transform 250ms ease;
-    &:hover {
-        transform: scale(.95);
+    img {
+        transition: transform 250ms ease;
+    }
+    &:hover img {
+        transform: scale(1.15);
+    }
+}
+.o_carousel_multiple_rows {
+    .row {
+        padding: 1rem;
+    }
+}
+
+// Cover image
+img.o_img_product_cover {
+    object-fit: cover;
+}
+// Force the image to have a square ratio
+img.o_img_product_square {
+    aspect-ratio: 1 / 1;
+}
+
+.o_card_group {
+    & > .row {
+        padding-left: $grid-gutter-width / 2;
+        padding-right: $grid-gutter-width / 2;
+    }
+    & > .row > div {
+        padding: 0 !important;
+    }
+    & > .row > div:first-child > .o_carousel_product_card {
+        border-left: $card-border-width solid $card-border-color !important;
+    }
+    & > .row:first-child .o_carousel_product_card {
+        border-top: $card-border-width solid $card-border-color !important;
+    }
+    & > .row:first-child > div:first-child .o_carousel_product_card {
+        @include border-top-start-radius($border-radius-lg !important);
+    }
+    & > .row:first-child > div:last-child .o_carousel_product_card {
+        @include border-top-end-radius($border-radius-lg !important);
+    }
+    & > .row:last-child > div:first-child .o_carousel_product_card {
+        @include border-bottom-start-radius($border-radius-lg !important);
+    }
+    & > .row:last-child > div:last-child .o_carousel_product_card {
+        @include border-bottom-end-radius($border-radius-lg !important);
```

Comment:
the `&` are no needed here

Reply:
several selectors can be grouped here... (e.g. `> .row:first-child >`, `> .row`)

Reply:
```scss
.o_card_group {
    > .row {
        padding-left: $grid-gutter-width / 2;
        padding-right: $grid-gutter-width / 2;
        > div {
            padding: 0 !important;
            &:first-child > .o_carousel_product_card{
                border-left: $card-border-width solid $card-border-color !important;
            }
        }
        &:first-child {
            .o_carousel_product_card {
                border-top: $card-border-width solid $card-border-color !important;
            }
            > div {
                &:first-child .o_carousel_product_card {
                    @include border-top-start-radius($border-radius-lg !important);
                }
                &:last-child .o_carousel_product_card {
                    @include border-top-end-radius($border-radius-lg !important);
                }
            }
        }
        &:last-child {
            > div {
                &:first-child .o_carousel_product_card {
                    @include border-bottom-start-radius($border-radius-lg !important);
                }
                &:last-child .o_carousel_product_card {
                    @include border-bottom-end-radius($border-radius-lg !important);
                }
            }
        }
    }
}
```

Would this be better? What do you think?

Reply:
> ```css-scss
> .o_card_group {
>     > .row {
>         padding-left: $grid-gutter-width / 2;
>         padding-right: $grid-gutter-width / 2;
>         > div {
>             padding: 0 !important;
>             &:first-child > .o_carousel_product_card{
>                 border-left: $card-border-width solid $card-border-color !important;
>             }
>         }
>         &:first-child {
>             .o_carousel_product_card {
>                 border-top: $card-border-width solid $card-border-color !important;
>             }
>             > div {
>                 &:first-child .o_carousel_product_card {
>                     @include border-top-start-radius($border-radius-lg !important);
>                 }
>                 &:last-child .o_carousel_product_card {
>                     @include border-top-end-radius($border-radius-lg !important);
>                 }
>             }
>         }
>         &:last-child {
>             > div {
>                 &:first-child .o_carousel_product_card {
>                     @include border-bottom-start-radius($border-radius-lg !important);
>                 }
>                 &:last-child .o_carousel_product_card {
>                     @include border-bottom-end-radius($border-radius-lg !important);
>                 }
>             }
>         }
>     }
> }
> ```
> 
> Would this be better? What do you think?

yes it's better like that ;)